python/ray/data/dataset.py

```diff
@@ -4131,6 +4131,10 @@ def get_internal_block_refs(self) -> List[ObjectRef[Block]]:
         self._synchronize_progress_bar()
         return blocks
 
+    @Deprecated(
+        message="Dataset is lazy by default, so this conversion call is no longer "
+        "needed and this API will be removed in future release"
```

Comment:
Nits (grammar): 

- "Datasets is lazy ..."
- "in a future release"

Reply:
Done (the `Dataset` here is referring to this class not the library we provide, so incline to keep it as `Dataset`).